packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -132,7 +132,8 @@ function getLocalReporter({ activity, reporter }) {
 function extendErrorIdWithPluginName(pluginName, errorMeta) {
   if (typeof errorMeta === `object`) {
     const id = errorMeta && errorMeta[`id`]
-    if (id) {
+    const isPrefixed = id.includes(`${pluginName}_`)
+    if (id && !isPrefixed) {
```

Comment:
If `id` was falsy then the above line will have crashed. Guess a test for this is hard?

Reply:
Moved the check a bit above